src/css/index.css

```diff
@@ -0,0 +1,143 @@
+/* =============================================================================
```

Comment:
This should also get renamed to nav.scss or something like that, then imported in index.scss so that webpack picks up on it.  You can see from the review app that's it's not getting built and included right now.